storage/database/db_manager.go

```diff
@@ -77,6 +77,7 @@ type DBManager interface {
 	WriteTd(hash common.Hash, number uint64, td *big.Int)
 	DeleteTd(hash common.Hash, number uint64)
 
+	ReadReceipt(txHash common.Hash) (*types.Receipt, common.Hash, uint64, uint64)
```

Comment:
```suggestion
	ReadReceipt(hash common.Hash) (*types.Receipt, common.Hash, uint64, uint64)
```

Reply:
@jeonghonah This is intended to distinguish `txHash` from `blockHash`